pkg/model/dev_test.go

```diff
@@ -92,7 +93,9 @@ swap:
     container: core
 mount:
   source: /Users/fernandomayofernandez/PycharmProjects/codescope-core
-  target: /app`)
+  target: /app
+scripts:
+  run: "uwsgi --gevent 100 --http-socket 0.0.0.0:8000 --mount /=codescope:app --python-autoreload 1"`)
```

Comment:
can we have another example?

Reply:
This is for a test, but what other example did you have in mind?